docs/plugins/contributors/fetchedUsers.json

```diff
@@ -433,5 +418,20 @@
     "avatar_url": "https://avatars.githubusercontent.com/u/129299274?v=4",
     "position": "",
     "github": "https://github.com/jozef-mikusinec-kiwicom"
+  },
+  {
+    "id": 158243242,
+    "name": null,
+    "username": "devin-ai-integration[bot]",
```

Comment:
😆

Reply:
We should filter it somehow. (or even get rid of this contributors page at all 🫣)

Reply:
It's merged. Do we need to close this PR? 🤔

Reply:
Yes. The script should run again soon and not trigger this change